DAO.sol

```diff
@@ -389,6 +389,9 @@ contract DAO is DAOInterface, Token, TokenSale {
             throw;
         }
 
+        if (_debatingPeriod > 8 weeks)
```

Comment:
What purpose does this max debating period serve? Perhaps also add a comment here explaining why?